code/controllers/subsystem/lobotomy_corp.dm

```diff
@@ -165,7 +182,7 @@ SUBSYSTEM_DEF(lobotomy_corp)
 		return FALSE
 	next_ordeal = pick(available_ordeals)
 	all_ordeals[next_ordeal_level] -= next_ordeal
-	next_ordeal_time = qliphoth_state + (next_ordeal_level * 2) + rand(1,3)
+	next_ordeal_time = qliphoth_state*1.5 + (next_ordeal_level * 2) + rand(1,3) // Increased
```

Comment:
Why?

Reply:
While testing it ordeals came WAY too fast, this is something I'm testing to see how people feel about it. Last time they simply didn't get enough abnos fast enough to be reasonable.